src/SensioLabs/Connect/Api/Entity/AbstractEntity.php

```diff
@@ -137,6 +137,10 @@ public function __call($name, $arguments)
         $method = substr($name, 0, 3);
         $property = lcfirst(substr($name, 3));
 
+        if (in_array($property, ['club', 'clubsUrl', 'project', 'projectsUrl'])) {
+            trigger_error(sprintf('The method "%s()" is deprecated since Connect 4.2.4. Will be removed in 5.0.', $name), E_USER_DEPRECATED);
```

Comment:
`[...] since Connect 4.3 and will [...]`

Reply:
Fixed.